hbase-server/src/test/java/org/apache/hadoop/hbase/mob/TestMobFileName.java

```diff
@@ -47,6 +47,7 @@
   private Date date;
   private String dateStr;
   private byte[] startKey;
+  private String regionName = "region";
 
```

Comment:
need a test for parsing MobFileName when a mob file doesn't have an encoded region name.

Reply:
I did not find a single usage of this class in the code which would benefited from this test case. Parsing file name into instance of MobFileName is used only in two test cases.